q2_diversity_lib/plugin_setup.py

```diff
@@ -0,0 +1,49 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2018-2019, QIIME 2 development team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file LICENSE, distributed with this software.
+# ----------------------------------------------------------------------------
+
+import q2_diversity_lib
+from qiime2.plugin import (Plugin, Citations, Properties)
+
+from q2_types.feature_table import (FeatureTable, Frequency)
+from q2_types.tree import Phylogeny, Rooted
+from q2_types.sample_data import AlphaDiversity, SampleData
+
+citations = Citations.load('citations.bib', package='q2_diversity_lib')
+plugin = Plugin(
+    name='diversity-lib',
+    version=q2_diversity_lib.__version__,
+    website='https://github.com/qiime2/q2-diversity-lib',
+    short_description='Utility exposing diversity metrics as actions',
```

Comment:
How about "Plugin for computing community diversity." ? I am trying to come up with something that pairs nicely with q2-diversity's short desc ("Plugin for exploring community diversity.").